src/Docker.PowerShell/Docker.psm1

```diff
@@ -0,0 +1,31 @@
+#
+# Script module for module 'PSScriptAnalyzer'
+#
+
+# Clear PSDefaultParameterValues in the module scope and enable strict mode
+$PSDefaultParameterValues.Clear()
+Set-StrictMode -Version Latest
+
+# Set up some helper variables to make it easier to work with the module
+$PSModule = $ExecutionContext.SessionState.Module
+$PSModuleRoot = $PSModule.ModuleBase
+
+# Import the appropriate nested binary module based on the current PowerShell version
+$binaryModuleRoot = $PSModuleRoot
+
+
+if (($PSVersionTable.Keys -contains "PSEdition") -and ($PSVersionTable.PSEdition -ne 'Desktop')) {
```

Comment:
Just write `$PSVersionTable["PSEdition"] -ne "Desktop"`. This works even with strict mode.